index.html

```diff
@@ -185,6 +185,149 @@ <h1>Terminology</h1>
+did:example:123?service=files&amp;relativeRef=/resume.pdf
+	</pre>
+
+	<p>
+		Some DID parameters are completely independent of of any specific <a>DID
+		method</a> and function the same way for all <a>DIDs</a>. Other DID parameters
+		are not supported by all <a>DID methods</a>. Where optional parameters are
+		supported, they are expected to operate uniformly across the <a>DID methods</a>
+		that do support them. The following table provides common DID parameters that
+		function the same way across all <a>DID methods</a>. Support for all
+		<a href="#did-parameters">DID Parameters</a> is OPTIONAL.
+	</p>
+
+	<table class="simple">
+		<thead>
+		<tr>
+			<th>
+				Parameter&nbsp;Name
+			</th>
+			<th>
+				Description
+			</th>
+		</tr>
+		</thead>
+
+		<tbody>
+		<tr>
+			<td>
+				<code><a>service</a></code>
+			</td>
+			<td>
+				Identifies a service from the <a>DID document</a> by service ID.
+				If present, the associated value MUST be an <a
+					data-lt="ascii string">ASCII string</a>.
+			</td>
+		</tr>
+		<tr>
+			<td>
+				<code>relativeRef</code>
+			</td>
+			<td>
+				A relative <a>URI</a> reference according to <a
+					data-cite="RFC3986#section-4.2">RFC3986 Section 4.2</a> that identifies a
+				<a>resource</a> at a <a>service endpoint</a>, which is selected from a <a>DID
+				document</a> by using the <code>service</code> parameter.
+				If present, the associated value MUST be an <a
+					data-lt="ascii string">ASCII string</a> and MUST use percent-encoding for
+				certain characters as specified in <a data-cite="RFC3986#section-2.1">RFC3986
+				Section 2.1</a>.
+			</td>
+		</tr>
+		<tr>
+			<td>
+				<code>versionId</code>
+			</td>
+			<td>
+				Identifies a specific version of a <a>DID document</a> to be resolved (the
+				version ID could be sequential, or a <a>UUID</a>, or method-specific).
+				If present, the associated value MUST be an <a
+					data-lt="ascii string">ASCII string</a>.
+			</td>
+		</tr>
+		<tr>
+			<td>
+				<code>versionTime</code>
+			</td>
+			<td>
+				Identifies a certain version timestamp of a <a>DID document</a> to be resolved.
+				That is, the <a>DID document</a> that was valid for a <a>DID</a> at a certain
+				time. If present, the associated value
+				MUST be an <a data-lt="ascii string">ASCII string</a> which is a valid XML
+				datetime value, as defined in section 3.3.7 of <a
+					href="https://www.w3.org/TR/xmlschema11-2/">W3C XML Schema Definition Language
+				(XSD) 1.1 Part 2: Datatypes</a> [[XMLSCHEMA11-2]]. This datetime value MUST be
+				normalized to UTC 00:00:00 and without sub-second decimal precision.
+				For example: <code>2020-12-20T19:17:47Z</code>.
+			</td>
+		</tr>
+		<tr>
+			<td>
+				<code>hl</code>
+			</td>
+			<td>
+				A resource hash of the <a>DID document</a> to add integrity protection, as
+				specified in [[?HASHLINK]]. This parameter is non-normative.
+				If present, the associated value MUST be an
+				<a data-lt="ascii string">ASCII string</a>.
```

Comment:
Not for this PR, but I wonder if we should just define hashlinks in this spec given that we have Multihash defined in CID now?